### PR TITLE
Added PowerShell examples to compliment the existing VBScript

### PR DESCRIPTION
**Pull request by ChrisKibble**
Included PowerShell examples for all code blocks where VBScript is there.  @aczechowski , this should be included in the most impactful submission because PowerShell > VBScript :-)

#MMS2019Docathon

### CODE REVIEW

**Comment by PRMerger6** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @ChrisKibble for the contribution! I'll review after the end of the contest.

#hold-off
(reviewers, please don't edit - I'll take care of it.)

File: sccm/apps/deploy-use/create-applications.md [hunk 1]
```diff
@@ -330,6 +330,9 @@ Continue to the next section on using a custom script as a detection method. Or
 
 Configuration Manager checks the results from the script. It reads the values written by the script to the standard output (STDOUT) stream, the standard error (STDERR) stream, and the exit code. If the script exits with a non-zero value, the script fails, and the application detection status is *Unknown*. If the exit code is zero, and STDOUT has data, the application detection status is *Installed*.
 
+> [!TIP]
+> This may not be intuitive for new Configuration Manager administrators.  When writing your detection script you cannot return any output unless you want to return that the application is installed.  See examples below the tables for help.
```
Comment (aczechowski) [inline sccm/apps/deploy-use/create-applications.md]: I suggest rewording a bit for style, let me know if it still means the same as your intent 😄 

```suggestion
> When writing a detection script, you can only return output (data in STDOUT) when it returns a zero exit code. The zero exit code means it detects the application as installed. For more information, see the following examples.  
```

Comment (ChrisKibble) [reply sccm/apps/deploy-use/create-applications.md]: Hi @aczechowski, still feels a bit confusing to me, how about somethinglike:

> When writing a detection script, if you return a zero exit code but don't return output (data in STDOUT), the application will not be detected as installed.  For more information, see the following examples.

Comment (aczechowski) [reply sccm/apps/deploy-use/create-applications.md]: That looks great, thanks @ChrisKibble (sorry for the slow response, been busy post-MMS with 1905 tech preview! 😮 💥 )

**Comment by PRMerger16** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/apps/deploy-use/create-applications.md (hunk 1 above)
Comment (ChrisKibble) [reply sccm/apps/deploy-use/create-applications.md]: Awesome, change made in patch-15!

Comment (ChrisKibble) [reply sccm/apps/deploy-use/create-applications.md]: And good luck with 1905 👍

**Comment by aczechowski** [comment]
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *should* be Tuesday, 28 May (due to the US holiday on Monday) around 11:00 Pacific time. No further action needed from you.
